src/app/components/ContactGroupsModal.js

```diff
@@ -31,6 +32,12 @@ const ContactGroupsTable = ({ onClose }) => {
     const api = useApi();
     const { call } = useEventManager();
 
+    const [list, setContactGroups] = useState(contactGroups || []);
```

Comment:
I prefer if you do `const [contactGroups = []] = useContactGroups`. @mmso needs to confirm.

Reply:
You can assume that contactGroups will always be loaded at this stage so no need to do `|| []`